### PR TITLE
use constants instead of hard-coded values in audio-maintainer and http-external-event-handler

### PR DESCRIPTION
**Pull request by skrashevich**

### CODE REVIEW

File: frigate/events/audio.py [hunk 1]
```diff
@@ -199,7 +199,7 @@ def detect_audio(self, audio) -> None:
                     continue
 
                 if score > dict((self.config.audio.filters or {}).get(label, {})).get(
-                    "threshold", 0.8
+                    "threshold", AUDIO_MIN_CONFIDENCE
```
Comment (NickM-27) [inline frigate/events/audio.py:202]: This is incorrect, 0.8 is the default min confidence for a filter, not 0.5 which is the min confidence for any audio detection

**Review by NickM-27 — CHANGES_REQUESTED** [review]

File: frigate/events/audio.py (hunk 1 above)
Comment (skrashevich) [reply frigate/events/audio.py:202]:
```
class AudioFilterConfig(FrigateBaseModel):
    threshold: float = Field(
        default=0.8,
        ge=AUDIO_MIN_CONFIDENCE,
        lt=1.0,
        title="Minimum detection confidence threshold for audio to be counted.",
    )
```
0.5<0.8

Comment (NickM-27) [reply frigate/events/audio.py:202]: > 0.5<0.8

Yes exactly, which is why this is incorrect. Not only does it not match the docs / default config, it is being more permissive than the default.

Comment (skrashevich) [reply frigate/events/audio.py:202]: hm. I'll re-check this

**Pull request closed without merging**